tools/config_updater.py

```diff
@@ -560,11 +560,55 @@ def ConfigureEmails(config):
   config.Set("Monitoring.emergency_access_email", emergency_email)
 
 
+def ConfigureUser(config):
+  success = False
+  print
```

Comment:
This was just there to add a newline for visual niceness during config.  I'm not a fan of doing it this way, but wasn't sure where else to put it.  Should I just add a leading \n to the RetryQuestion() call?